app/mocks/handlers/handlers.ts

```diff
@@ -1,5 +1,5 @@
 import { rest } from 'msw';
-const URL = window.ENV.API_HOST;
+const URL = process.env.API_HOST;
```

Comment:
why this change?
also we'll have to move it to the `url.constants.ts` file. I'll do that in my PR

Reply:
removed this file now, the URL was getting called when we were mocking msw server on jest and since jest runs in a node environment we do not have a window object there, but this change was breaking development as we require the URL on client because we have a request running on client, so I have created separate files